facdb/sql/usdot_airports.sql

```diff
@@ -2,31 +2,37 @@ DROP TABLE IF EXISTS _usdot_airports;
 SELECT
     uid,
     source,
-    fac_name as facname,
-    NULL as addressnum,
-    NULL as streetname,
-    NULL as address,
+    name as facname,
+    parsed_hnum as addressnum,
+    parsed_sname as streetname,
+    cleaned_address as address,
     city,
-    NULL as zipcode,
+    zipcode as zipcode,
     county as boro,
     NULL as borocode,
     NULL as bin,
     NULL as bbl,
-    fac_type as factype,
+    facility_type as factype,
     'Airports and Heliports' as facsubgrp,
     (CASE
-        WHEN owner_type = 'Pr' THEN fac_name
+        WHEN ownership= 'PR' THEN name
         ELSE 'Public'
     END) as opname,
     (CASE
-        WHEN owner_type = 'Pr' THEN 'Non-public'
+        WHEN ownership = 'PR' THEN 'Non-public'
         ELSE 'Public'
     END) as opabbrev,
     'USDOT' as overabbrev,
     NULL as capacity,
     NULL as captype,
-    wkt::geometry as wkb_geometry,
-    geo_1b,
+    --- wkt::geometry as wkb_geometry,
+    NULL as wkb_geometry,
+    (CASE
+        WHEN facility_type = 'AIRPORT' THEN geo_1a
+        WHEN name = 'NYPD AIR OPERATIONS (FLOYD BENNETT FLD)' THEN geo_1a
+        ELSE geo_1b :: VARCHAR
+        END
+    ) as geo_1b,
```

Comment:
This is confusing to me, seems like there should be a better way than re-casting 1A as 1B